01-basics/03-MarkedList/index.html

```diff
@@ -1,30 +1,33 @@
 <!DOCTYPE html>
 <html lang="ru">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="/assets/styles/index.css" />
-    <title>MarkedList</title>
-    <style>
-      .marked {
-        color: #efc14e;
-      }
-    </style>
-  </head>
-  <body>
-    <div id="app" style="margin-top: 2rem">
-      <div class="container">
-        <div class="form-group">
-          <input type="search" />
-        </div>
-        <ul>
-          <li>Email</li>
-          <li class="marked">Email</li>
-          <li>...</li>
-          <li>Email</li>
-        </ul>
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="stylesheet" href="/assets/styles/index.css" />
+  <title>MarkedList</title>
+  <style>
+    .marked {
+      color: #efc14e;
+    }
+
+  </style>
+</head>
+
+<body>
+  <div id="app" style="margin-top: 2rem">
+    <div class="container">
+      <div class="form-group">
+        <input v-model="searchString" type="search" />
       </div>
+      <ul>
+        <template v-for="item in markedEmails">
+          <li :class="{marked: item.marked}">{{item.email}}</li>
+        </template>
```

Comment:
`<template v-if>` and `<template v-for>` are used only to wrap several nodes for `v-if`/`v-for` directives. With one node it is redudant - `v-for` can be used on the node directly.
```html
<li v-for="item in markedEmails" :class="{marked: item.marked}">{{item.email}}</li>
```